Junior (#28203)
ocaml/opam-repository: CONTRIBUTING.md

Contributing to the OCaml opam repository
-----------------------------------------

> Contributions under some form of new packages, issue reports and pull
> requests to fix and enhance the quality of packages are always

Reviewer: I think the right idiom for this is probably

```suggestion
Contributions in the form of new packages, issue reports and pull
requests to fix and enhance the quality of packages are always
```
